static/app/stores/pageFiltersStore.tsx

```diff
@@ -3,19 +3,21 @@ import Reflux from 'reflux';
 
 import PageFiltersActions from 'sentry/actions/pageFiltersActions';
 import {getDefaultSelection} from 'sentry/components/organizations/pageFilters/utils';
-import {Organization, PageFilters} from 'sentry/types';
+import {Organization, PageFilters, PinnedPageFilter} from 'sentry/types';
 import {isEqualWithDates} from 'sentry/utils/isEqualWithDates';
 
 import {CommonStoreInterface} from './types';
 
 type State = {
   selection: PageFilters;
+  pinnedSelection: Set<PinnedPageFilter>;
```

Comment:
how do we feel about the name `pinnedFilters`? I think it makes more sense since this represents a set of the pinned page filters. `pinnedSelection` almost makes this var seem like another `selection` object rather than just a set of pinned categories/dropdowns.

Reply:
Yeah good idea